runtime/amplitude/src/lib.rs

```diff
@@ -1244,6 +1286,7 @@ construct_runtime!(
 		Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 16,
 		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 17,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 18,
+		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 92,
```

Comment:
Seems like `22` is also not used anywhere. How about we just use that one then? Or did we use it before for another pallet that we removed in the meantime?
I'm actually not sure if it would pose a problem if we re-assign indices that were previously used... do you think you could find out @b-yap?

Reply:
Thanks for sharing [this](https://substrate.stackexchange.com/a/10429/2473).
Let's go for `22` then and put it below `ChildBounties`

Reply:
done